src/main/resources/templates/snowflake/snowflake-schema-evolution/snowpipe-redefinition.sql.ssp

```diff
@@ -0,0 +1,69 @@
+#import(io.phdata.streamliner.schemadefiner.model.Snowflake)
+<%@ val configuration: io.phdata.streamliner.schemadefiner.model.Configuration %>
+<%@ val table: io.phdata.streamliner.schemadefiner.model.SnowflakeTable %>
+<%@ val typeMapping: Map[String, Map[String, String]]%>
+#{
+    val destination = configuration.destination.asInstanceOf[Snowflake]
+    val sourceColumnStr = table.sourceColumnConversion(typeMapping)
+}#
```

Comment:
should I put a check here tableDiff.allChangesAreCompatible() ?
Because even for modify columns also these scripts are generated. 
Currently we are only processing add columns and for other operations throwing error message.

Reply:
Seems like yes we would only want to output this when we are doing a all changes are compatible.

Reply:
updated